cms/envs/common.py

```diff
@@ -465,6 +467,8 @@
 USE_I18N = True
 USE_L10N = True
 
+STATICI18N_ROOT = PROJECT_ROOT / "static"
```

Comment:
cool, are these generated as a part of collectstatic or is there a new management command that needs to be run?  Sounds like it's done as a part of an existing command since it's working on sandboxes but just wanted to verify.